news/2020-31-12-mappers-report-fall.md

```diff
@@ -0,0 +1,265 @@
+
+One of his most solidified maps in his mapping discography is [Konami Amusement (Wakita Jun) - Vide Nostalgia](https://osu.ppy.sh/beatmapsets/1197900#mania/2530522), which not only displays some excellent timing craftsmanship, but as well as heavy PR patterning whilst maintaining mapping standards that fit along with the song beautifully. The map also features a constant 3/4 timing signature that can somewhat throw players off from your normal 4/4 timing signature that many are accustomed to, making this quite a challenge to acc. The mapset as a whole resembles MJH’s current mapping style perfectly, which is why it holds such an important place in his mapping discography. 
+
+Another mapset of his that is rather new than some of older mapsets, but still an amazing mapset anyhow is [Akaru muu - Choukousoku henbyoushi no alien](https://osu.ppy.sh/beatmapsets/1324800#mania/2749712), which somewhat resembles a lot of big and notable features that Vide Nostalgia has, but turned up a couple notches. Easily one of his more complex and interesting maps, the map features very odd but complexing time signatures that’ll definitely catch many players off guard, whilst capturing the song through pretty complex rhythms and structures. He does all of this whilst also maintaining PR and heavy repetition when the song calls for it, making this one hell of a challenge to FC and acc for many players. Definitely give this a try if you’re looking for a challenge. 
+
+We hope to see MJHs make more maps in the future as it’s pretty clear he’s making a name for himself in the 5K community thus far, and hopefully we can see him rank some maps whilst he’s at it!
+
+## <a id="mapping-contests"></a>Mapping Contests
+
+Did someone ask for contests? No? Ahh well, you get them anyways! As always we inform you about any interesting mapping related contests going on. Even if you do not plan to participate in them, you might want to check out if any of them has an open voting phase going on! Don't forget: Every vote counts!
+
+And don't forget. Should you have any interesting contests planned or know about, let us know! We would love to give them more attention in our news post!
```

Comment:
"Should you have any interesting contests planned or know about" >> "Should you have or know about any interesting contests planned,"